RentACar/RentACar/CRUD/CarCRUD.cs

```diff
@@ -1,69 +1,66 @@
-﻿using System;
-using System.IO;
-using System.Text.Json;
+﻿using RentACar.Helpers;
 using System.Collections.Generic;
 using System.Linq;
-
+using RentACar.CRUD;
 
 namespace RentACar
 {
-    public class CarCRUD
+    public class CarCRUD : ICarCRUD
     {
-        private readonly string _jsonFile = @"..\..\..\CRUD\Cars.json";
+        private readonly string _jsonFile = @"..\..\..\Data\Cars.json";
```

Comment:
we can improve this, check https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-5.0 and let me know

Reply:
I was really trying last night and I'm a bit confused about (in general) where to do all of that.
But let me keep trying now with fresh air, and if I can't, I'll try to ask the right questions.